image/png.ksy

```diff
@@ -304,3 +359,10 @@ enums:
     1: meter
   compression_methods:
     0: zlib
+  dispose_op_values:
+   0: none
+   1: background
+   2: previous
+  blend_op_values:
+   0: source
+   1: over
```

Comment:
Nit: please use 2-space indent to these values.

Reply:
oh, oops. I will fix that.